script/github-actions/run-cypress-tests.js

```diff
@@ -26,16 +27,21 @@ const batch = tests
   .slice(step * divider, (step + 1) * divider)
   .join(',');
 
-if (longestTestIsPresent && step === lastStep) {
-  const status = runCommandSync(
-    `CYPRESS_EVERY_NTH_FRAME=1 yarn cy:run --browser chrome --headless --reporter cypress-multi-reporters --reporter-options "configFile=config/cypress-reporters.json" --spec 'src/applications/**/all-claims.cypress.spec.js' --env app_url=${appUrl}`,
-  );
-  process.exit(status);
-} else if (batch !== '') {
-  const status = runCommandSync(
-    `CYPRESS_EVERY_NTH_FRAME=1 yarn cy:run --browser chrome --headless --reporter cypress-multi-reporters --reporter-options "configFile=config/cypress-reporters.json" --spec '${batch}' --env app_url=${appUrl}`,
-  );
-  process.exit(status);
-} else {
-  process.exit(0);
+let status = null;
+// const runTestsInLoopUpTo = isStressTest ? 25 : 1;
```

Comment:
Are these testing changes that were left commented out?

Reply:
Yes, I'll update those now. Thanks!